editor/project_settings_editor.cpp

```diff
@@ -1127,7 +1127,12 @@ void ProjectSettingsEditor::_translation_res_option_changed() {
 	ERR_FAIL_COND(!remaps.has(key));
 	PoolStringArray r = remaps[key];
 	ERR_FAIL_INDEX(idx, r.size());
-	r.set(idx, path + ":" + langs[which]);
+	if(filtered_locales_idxs_remap.size()>0){
+		r.set(idx, path + ":" + langs[filtered_locales_idxs_remap[which]]);
+    }
+    else{
+        r.set(idx, path + ":" + langs[which]);
+    }
```

Comment:
The style is completely wrong here. See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html